content/en/getting_started/synthetics/private_location.md

```diff
@@ -19,19 +19,21 @@ The access to this feature is restricted - if you don't have access, reach out t
 
 ## Overview
 
-Private locations allow you to **monitor internal-facing applications or any private URLs** that aren’t accessible from the public internet. They can also be used to:
+Private locations allow you to **monitor internal-facing applications** or **private URLs** that aren’t accessible from the public internet. 
 
-* **Create custom locations** in areas that are mission-critical to your business.
-* **Verify application performance in your internal CI environment** before you release new features to production with [Synthetic CI test integration][1].
-* **Compare application performance** from both inside and outside your internal network.
+You can also:
 
-Private locations come as Docker containers that you can install wherever makes sense inside of your private network. Once created and installed, you can assign [Synthetic tests][2] to your private location just like you would with any regular managed location.
+- Create custom locations in mission-critical areas of your business.
```

Comment:
I liked how the bold we were using before made the four main use cases very clear 😬

Reply:
Bringing back the bold!